pallets/automation-time/Cargo.toml

```diff
@@ -41,6 +41,9 @@ pallet-timestamp = { git = "https://github.com/paritytech/substrate", default-fe
 ## Moonbeam Dependencies
 pallet-parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "oak-polkadot-v0.9.23" }
 
+## Local
+pallet-automation-time-rpc-runtime-api = { path = "./rpc/runtime-api", default-features = false }
```

Comment:
This is the only part of this pr that I'm not totally sure about. I think we're fine importing this here and the compiler should have yelled at us if we were doing something really bad, but std/wasm issues have surprised me in the past. I think we're fine here, but flagging in case anyone has a second opinion.